### DISCUSSION TITLE
Zero Regularization Multiplier

### DISCUSSION THREAD

**Discussion by lalitkumarj** [cvxpy/cvxpy#1515] [Q&A]
Hi

I have been using CVXPY (with SCS) to solve a rank-constrained problem. To calibrate against least squares, I considered two different objectives:
 
``objective_1 = cp.Minimize(cp.sum_squares(X@bet + R@alph + U - Y)/n + lam*cp.normNuc(Gam))``
and 
 ``objective_2 = cp.Minimize(cp.sum_squares(X@bet + R@alph + U - Y)/n )``

When I set lam=0 and run objective_1 I can get very different results from objective_2 (in the sense that the resulting MSE of the output solutions are off by a factor of 2 different). Is there any reason for why I might expect wildly different answers? If more code helps to reproduce the issue, I am happy to attach it!

**Comment by SteveDiamond** [comment]
Are you sure that the same solver is being used in both cases? By default SCS will be used for the first objective, and OSQP for the second.

**Reply by rileyjmurray** [reply]
@lalitkumarj to get an answer to Steven's question, try calling ``prob.solve(verbose=True)`` and then pasting the input on this discussion.
